expression/builtin_json.go

```diff
@@ -237,6 +238,45 @@ func JSONMerge(args []types.Datum, sc *variable.StatementContext) (d types.Datum
 	return
 }
 
+// JSONObject creates a json from an ordered key-value slice.
+func JSONObject(args []types.Datum, sc *variable.StatementContext) (d types.Datum, err error) {
```

Comment:
in `expression/builtin.go`, we defined this function with 2 arguments at least. Here I think add more documents is better.